Add files via upload

DimasBykov opened the pull request.

DimasBykov committed:
Add files via upload

alexnalivayko commented:
Системные файлы (для IDE - com.netcracker.iml, target папку с скомпилированными файлами) не нужно добавлять к коммиту. Давай уберем эти ненужные файлы со след. твоими изменениями

DimasBykov committed:
Add files via upload

Added "exit" command and  CSV to XML saver with the option to save both files or delete CSV.
.

alexnalivayko commented on src/main/java/view/View.java:
```diff
@@ -0,0 +1,82 @@
+	public void active() throws IOException {
+		int act = in.nextInt();
+
+		if (act<=5){
```
Давай лучше сделаем **_default_**-case в **_swith_**'e с выводом информации пользователю, что программа не знает такую операцию, вместо проверки на числовое ограничение. 
Попробуй набрать отрицательное число (например, -3), и ты поймешь, что я имею ввиду

alexnalivayko commented on src/main/java/Main.java:
```diff
@@ -0,0 +1,17 @@
+
+import view.View;
```
Давай перенесем все твои новые классы/пакеты в пакет `com.netcracker.edu`

alexnalivayko commented on pom.xml:
```diff
@@ -1,26 +1,57 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>oblako</groupId>
```
Затерлись предыдущие значения тегов `<groupId>, <artifactId>, <version>, <developers>`. Нужно вернуть + блок `<build>`, как правило, находится самым последнем в файле `pom.xml`.

alexnalivayko commented on com.netcracker.iml:
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Нужно удалить данный файл из репозитория

alexnalivayko commented on src/main/java/Main.java:
```diff
@@ -0,0 +1,17 @@
+
+public class Main {
+
+	public static void main(String[] args) throws IOException {
```
Давай сделаем так, что только методы класса Controller будут "выплевывать" исключения (по возможности). Во всех остальных классах мы будем их "ловить" через try/catch

alexnalivayko commented on src/main/java/view/View.java:
```diff
@@ -0,0 +1,82 @@
+			switch (act){
+
+				case 1:
+					System.out.println("Укажите путь и имя файла с указанием формата файла в формате: " +
```
Все подсказки для пользователя лучше помещать (в скобки)
"Укажите путь и имя файла (с указанием расширения, 'C:\Users\FileName.csv'):"

alexnalivayko commented on src/main/java/controller/Controller.java:
```diff
@@ -0,0 +1,122 @@
+		String [] record = "id,name,way,USD".split(",");
+		writer.writeNext(record);
+		writer.close();
+		System.out.println("Файл бы успешно создан\n");
```
был*

alexnalivayko commented on src/main/java/controller/Controller.java:
```diff
@@ -0,0 +1,122 @@
+	}
+
+	public void createFile(String line) throws IOException {
+		if (line.indexOf(".csv")==0){
```
С помощью данного метода удалось создать файлы с разными расширениями, помимо CSV:
![image](https://user-images.githubusercontent.com/23340099/49941576-5e2cad00-fefc-11e8-9dd8-3f91d1ca86ae.png)
Думаю, что ошибка в твоей проверки на расширение

alexnalivayko commented on src/main/java/view/View.java:
```diff
@@ -0,0 +1,82 @@
+	private Controller controller = new Controller();
+
+	public void hello() throws IOException {
+		System.out.println("\nДобро пожаловать! \n" +
```
"Добро пожаловать!" встречается при каждом появлении меню. Давай поздароваемся с пользователем только один раз, а все остальные попытки - выводить меню

DimasBykov committed:
Add files via upload

DimasBykov committed:
Delete Main.java

DimasBykov committed:
files

deleted useless files

DimasBykov committed:
final?

DimasBykov committed:
helpers

DimasBykov committed:
second part version 1.0

DimasBykov committed:
not final((

DimasBykov committed:
rm rf

DimasBykov committed:
seems like final

DimasBykov committed:
seems like final

DimasBykov committed:
seems like final
